src/components/RedirectToNextGenDialog.svelte

```diff
@@ -0,0 +1,59 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+ 
+  SPDX-License-Identifier: MIT
+ -->
+
+<script lang="ts">
+  import { t } from '../i18n';
+  import { tryLocalStorageGet, tryLocalStorageSet } from '../script/utils/local-storage';
+  import StandardDialog from './dialogs/StandardDialog.svelte';
+  import StandardButton from './StandardButton.svelte';
+
+  const seenRedirectDialogKey = 'seenRedirectDialog';
+  const seenRedirectDialogValue = 'true';
+
+  const setAsSeenRedirectDialog = () => {
+    tryLocalStorageSet(seenRedirectDialogKey, seenRedirectDialogValue);
+  };
+
+  const hasSeenRedirectDialog = () =>
+    tryLocalStorageGet(seenRedirectDialogKey) === seenRedirectDialogValue;
```

Comment:
If we follow the existing app structure I think we can use persistantWritable here instead of creating tryLocalStorage.

Reply:
I nearly suggested that... but we don't need the store behaviour and it might be nice if it didn't error on write. Though the rest of the code will have that issue.

Reply:
I updated it to use `persistantWritable` just to be consistent